react/features/base/media/components/AbstractVideoTrack.js

```diff
@@ -107,14 +112,21 @@ export default class AbstractVideoTrack extends Component {
 
         const stream
             = render ? videoTrack.jitsiTrack.getOriginalStream() : null;
+        const streamType = render ? videoTrack.videoType : null;
+
+        // Actual zoom is currently only enabled if the stream is a desktop
+        // stream.
+        const _zoomEnabled = zoomEnabled && streamType === 'desktop';
 
         return (
             <Video
                 mirror = { videoTrack && videoTrack.mirror }
                 onPlaying = { this._onVideoPlaying }
                 onPress = { onPress }
                 stream = { stream }
-                zOrder = { this.props.zOrder } />
+                type = { streamType }
```

Comment:
I don't see the `Video` prop `type` used anywhere (in `Video`). Which isn't surprising given that the stream's type only enables/disables the 'pinch to zoom' and that happened a few lines above.